java-google-style.xml

```diff
@@ -0,0 +1,279 @@
+<code_scheme name="GoogleStyle">
```

Comment:
This settings file seems a bit short, the original google java style for intellij has 475 lines, while this one has around 300.

The settings in pull request https://github.com/google/styleguide/pull/142 seems the most up-to-date one, please give it a try.